webpack.config.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    devtool: 'source-map',
+    entry: './samples/index.js',
+    output: {
+        path: __dirname + 'samples/public',
```

Comment:
Instead of string concatenation for paths, please use `path.resolve()` function here. See [this](https://github.com/wso2/carbon-dashboards/blob/v4.0.0.alpha4/samples/widgets/CpuUsage/webpack.config.js#L24)